ann_benchmarks/runner.py

```diff
@@ -263,12 +288,15 @@ def stream_logs():
     finally:
         container.remove(force=True)
 
+
 def _handle_container_return_value(return_value, container, logger):
     base_msg = 'Child process for container %s' % (container.short_id)
-    if type(return_value) is dict: # The return value from container.wait changes from int to dict in docker 3.0.0
-        error_msg = return_value['Error']
+    # The return value from container.wait changes from int to dict in docker 3.0.0
+    if type(return_value) is dict:
```

Comment:
I'm not sure if something has changed recently in the `docker-py` library but it looks like a dict is always returned now and it doesn't always contain an `Error` key.

Reply:
yeah the docker library has changed a few times with similar things, it's a bit annoying